qa/tasks/ceph_manager.py

```diff
@@ -1786,7 +1786,13 @@ def get_pg_stats(self):
         """
         out = self.raw_cluster_cmd('pg', 'dump', '--format=json')
         j = json.loads('\n'.join(out.split('\n')[1:]))
-        return j['pg_stats']
+        try:
+            new_stats = j['pg_map']['pg_stats']
+        except:
```

Comment:
please be specific that we are expecting `KeyError` here.

Reply:
yes, updated!